README.md

```diff
@@ -31,6 +31,12 @@ rust-openssl to a separate installation. OSX releases starting at 10.11, "El
 Capitan", no longer include OpenSSL headers which will prevent the `openssl`
 crate from compiling.
 
+Try the following commands to fix the issue (assumes Homebrew is used for package management):
+```
+brew install openssl
+brew link openssl --force
```

Comment:
I hear you about the --force part, that may be hard to make as a general recommendation. By `OPENSSL_LIB_DIR` do you mean setting them in Cargo.toml or somewhere like that or in bash profile? I am not experienced about the former, but the latter does not look to me any different than `brew link openssl --force`.

I am curious about your answer. According to that I will close this PR/